exercises/phone-number/examples/success-standard/src/Phone.hs

```diff
@@ -1,9 +1,18 @@
 module Phone (number) where
-import Data.Char (isDigit, isLetter)
+import Data.Char (isDigit)
 
 number :: String -> Maybe String
-number input
-  | any isLetter input = Nothing
+number input = clean input >>= check
```

Comment:
pointfree.io tells me this is `number = (check =<<) . clean` but I'm not sure I consider this better.